src/wallet/api/wallet2_api.h

```diff
@@ -772,18 +785,40 @@ struct WalletManager
      */
     virtual Wallet * recoveryWallet(const std::string &path, const std::string &memo, bool testnet = false, uint64_t restoreHeight = 0) = 0;
 
-   /*!
-    * \brief  recovers existing wallet using keys. Creates a view only wallet if spend key is omitted
-    * \param  path           Name of wallet file to be created
-    * \param  language       language
-    * \param  testnet        testnet
-    * \param  restoreHeight  restore from start height
-    * \param  addressString  public address
-    * \param  viewKeyString  view key
-    * \param  spendKeyString spend key (optional)
-    * \return                Wallet instance (Wallet::status() needs to be called to check if recovered successfully)
-    */
-    virtual Wallet * createWalletFromKeys(const std::string &path, 
+    /*!
```

Comment:
Looks like indentation was changed here too. It'd make sense to revert the whitespace change as it is unrelated to this particular change.

Reply:
Yes, one whitespace was added to the head of each line.

Reply:
the method L766-L775 is new. so is the method L788-L807. The comments for methods L777-L786 & L809-L827 have been modified ("deprecated" added).
i do not understand why you feel any changes in these areas are unrelated.

Reply:
They are unrelated because — in a perfect world — a PR should only include a _single compact change_; i.e., this PR should _only_ modify stuff which is absolutely crucial to meet its goal. It'd best to never adjusting indentation, even if it is code you're currently working on. I'm totally fine with you adding a few spaces, however one should try to do that in a separate pull request which only does this one thing: fixing inconsistent indentation.

Reply:
Please revert these whitespace changes.